src/applications/check-in/day-of/pages/Demographics.jsx

```diff
@@ -63,10 +63,6 @@ const Demographics = props => {
     ],
   );
 
-  if (!demographics) {
```

Comment:
NITPICK: Looks like we are losing the source of the error. Unless I am missing something, which I could be

Reply:
These missing data catches were a bit of un-needed code, now all error handling is happening at the API request. If we don't get this data it will be caught in request.